packages/core/src/DatumBuilders/DatumBuilder.NftCheck.class.ts

```diff
@@ -0,0 +1,47 @@
+import { Core, Data } from "@blaze-cardano/sdk";
+import { AssetAmount, IAssetAmountMetadata } from "@sundaeswap/asset";
+import {
+  NftCheckDatum,
+  TCheck,
+  TNftCheckDatum,
+} from "./ContractTypes/Contract.NftCheck";
+import { DatumBuilderCondition } from "./DatumBuilder.Condition.class";
```

Comment:
Make sure that all imports use module syntax (i.e. explicit file imports that end in `.js`)